app-emulation/wine-vanilla/files/wine-vanilla-6.12-winegcc-equals-args.patch

```diff
@@ -0,0 +1,13 @@
+https://bugs.winehq.org/show_bug.cgi?id=51413
```

Comment:
Do we need to do this for app-emulation/wine-staging-6.12 too?

Reply:
Yes.